### DISCUSSION TITLE
Group Meeting 1

### DISCUSSION THREAD

**Discussion by scottctd** [UofT-CS-Group/CSC110-Final-Project#6] [General]
- We need to have a group meeting to talk about the infrastructure of our program.
- A temporary timing is **Friday, Nov.5, 2021, 9:30 PM EDT**.
- Before the meeting, we need to make sure that most people have a vague idea about PyQt5 and the data sets we are going to use.
- If you have a time conflict, please leave it below this discussion and I will change our timing based on everyone's availability.
- Zoom Meeting: 233 666 8868.
- See you there!

**Comment by Lei-Tin** [comment]
"When to meet" is a good website to negotiate on the timing for meetings, but that's cool.

I'm ok with Friday 9:3 0EDT. 
I have ideas on the Data sets and a "Very very limited" idea on Pyqt. 

Scott big leg pls carry hard :3

**Comment by alyssamyli** [comment]
咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩咩

**Comment by scottctd** [comment]
UofT的网修好啦

很抱歉 这次出了挺多意外 我说的好多东西也可能太主观 不太符合咱现在该做的

如果有任何意见和idea，直接喷我就行，我挺希望有人能喷我，不是说吹我，因为我真的很慌，捧得太高会跌的最惨

[这次Group Meeting的Note点我](https://github.com/UofT-CS-Group/CSC110-Final-Project/blob/0ed10d5abdf89b0c1a1ad0ad04e2e6c48446fba3/GroupMeetings/1_20211105/group_meeting_1.md)

一个手画的visualization graph
![image](https://user-images.githubusercontent.com/35100768/140598745-2e3bc3da-02f6-40b3-840a-7d04612bda6d.png)

(Don't blame me for this, i'm totally a blockhead in drawing

![image](https://user-images.githubusercontent.com/35100768/140598756-aa27499d-938a-4bb6-aad9-7dbec5bc009d.png)

类的结构图这是

我一会会把data module的结构写一下pr一波，体验一下架构师的感觉哈哈哈哈哈
